data/strings/strings.txt

```diff
@@ -26894,3 +26894,381 @@
     vi = Cảm ơn bạn đã sử dụng bản đồ do cộng đồng xây dựng của chúng tôi!
     zh-Hans = 感谢您使用我们的社区地图！
     zh-Hant = 感謝您使用我們社區構建的地圖！
+
+  [app_tip_01]
+    comment = App tip #01
+    tags = android
+    en = With your donations and support, we can create the best maps app in the World!
```

Comment:
Let's simplify it a bit and remove app, translations should be better after that.
```suggestion
    en = With your donations and support, we can create the best maps in the World!
```

Reply:
Modified all translations of this tip, removing "app"...